app/admin/dashboard.rb

```diff
@@ -3,31 +3,21 @@
   menu priority: 1, label: proc{ I18n.t("active_admin.dashboard") }
 
   content title: proc{ I18n.t("active_admin.dashboard") } do
-    div class: "blank_slate_container", id: "dashboard_default_message" do
-      span class: "blank_slate" do
-        span I18n.t("active_admin.dashboard_welcome.welcome")
-        small I18n.t("active_admin.dashboard_welcome.call_to_action")
+
+    columns do
+      column do
+        panel "Info" do
+            @user_chapter = Chapter.where(id: current_admin_user.chapter_id)
+            @user_chapter.each do |c|
+                if c.is_active?
```

Comment:
You would have to manually make the `is_active?` method for this to work. That would look something like this.

```
def is_active?
  is_active
end

Your intuition is right though. It's idiomatic (but not required) in Ruby to end methods that return booleans with a question mark. That may be a bit superfluous at this time, so I would adjust it to just be `c.is_active` and it will work just fine!